src/components/oculus-touch-controls.js

```diff
@@ -81,6 +88,13 @@ module.exports.Component = registerComponent('oculus-touch-controls', {
     this.previousButtonValues = {};
     this.bindMethods();
     this.isControllerPresent = isControllerPresent; // to allow mock
+    this.getGamepadsByPrefix = getGamepadsByPrefix; // to allow mock
+  },
+
+  update: function (oldData) {
+    if ((oldData.emulated !== undefined) && (this.data.emulated !== oldData.emulated)) {
+      this.checkIfControllersPresent();
```

Comment:
in order for that line to be covered, the entity would have to have emulated be true right at initial definition, and the tests use a setup function where no attributes are provided initially.